package/MDAnalysis/core/AtomGroup.py

```diff
@@ -457,8 +457,84 @@
 # And the return route
 _SINGULAR_PROPERTIES = {v: k for k, v in _PLURAL_PROPERTIES.items()}
 
-_FIFTEEN_DEPRECATION = "This will be removed in version 0.15.0"
+_FIFTEEN_DEPRECATION = "This will be removed in version 0.16.0"
```

Comment:
Can you run a global replace to `_SIXTEEN_DEPRECATION`? In case you are using emacs and projectile the short cut it `C-p r`